src/language-js/print/call-arguments.js

```diff
@@ -253,21 +258,58 @@ function couldGroupArg(arg) {
   );
 }
 
-function shouldGroupLastArg(args) {
+function shouldGroupLastArg(args, callee, parentType) {
   const lastArg = getLast(args);
-  const penultimateArg = getPenultimate(args);
-  return (
-    !hasComment(lastArg, CommentCheckFlags.Leading) &&
-    !hasComment(lastArg, CommentCheckFlags.Trailing) &&
-    couldGroupArg(lastArg) &&
-    // If the last two arguments are of the same type,
-    // disable last element expansion.
-    (!penultimateArg || penultimateArg.type !== lastArg.type) &&
+
+  if (
+    hasComment(
+      lastArg,
+      CommentCheckFlags.Leading | CommentCheckFlags.Trailing
```

Comment:
@fisker This doesn't work as expected. Something is wrong inside `hasComment`.

Reply:
I was wrong. Actually, it makes sense. `CommentCheckFlags.Leading | CommentCheckFlags.Trailing` doesn't mean "leading or trailing". It means "leading and trailing".

Reply:
Maybe we can add conflicting flags check? eg `Leading`/`Trailing`/`Dangling` and `Dangling`/`Line`

Reply:
Yes, as an assertion removed at build time. Good idea.